src/locales/en-US/admin.ftl

```diff
@@ -145,18 +145,18 @@ configure-general-commentEditing-seconds = Seconds
 
 #### Closed Stream Message
 configure-general-closedStreamMessage-title = Closed comment stream message
-configure-general-closedStreamMessage-explanation = Write a message to appear after a story is closed for commenting.
+configure-general-closedStreamMessage-explanation = Write a message to appear when a story is closed for commenting.
 
 ### Organization
 configure-organization-name = Organization name
 configure-organization-nameExplanation =
   Your organization name will appear on emails sent by { -product-name } to your community and organization members.
 configure-organization-email = Organization email
 configure-organization-emailExplanation =
-  This email address will be used as in emails and across
-  the platform for community members to get in touch with
-  the organization should they have any questions about the
-  status of their accounts or moderation questions.
+  Open or close comment streams for new comments sitewide. When new comments
+  are turned off sitewide, new comments cannot be submitted, but existing
+  comments can continue to receive “Respect” reactions, be reported, and be
```

Comment:
Maybe leave the `"Respect"` out as it can be customised to anything?

Reply:
Nice catch! Fixing now